### PR TITLE
disable memsize logging by default

### PR DESCRIPTION
**Pull request by mikedoan**
disable memsize logging by default

### CODE REVIEW

File: cmd/harmony/main.go [hunk 1]
```diff
@@ -94,9 +94,10 @@ var (
 	// isArchival indicates this node is an archival node that will save and archive current blockchain
 	isArchival = flag.Bool("is_archival", false, "true means this node is a archival node")
 	//isNewNode indicates this node is a new node
-	isNewNode    = flag.Bool("is_newnode", false, "true means this node is a new node")
-	accountIndex = flag.Int("account_index", 0, "the index of the staking account to use")
-	shardID      = flag.Int("shard_id", -1, "the shard ID of this node")
+	isNewNode     = flag.Bool("is_newnode", false, "true means this node is a new node")
+	accountIndex  = flag.Int("account_index", 0, "the index of the staking account to use")
+	shardID       = flag.Int("shard_id", -1, "the shard ID of this node")
+	enableMemsize = flag.Bool("enableMemsize", false, "Enable memsize logging.")
```
Comment (rlan35) [inline cmd/harmony/main.go]: NIT: change to "enableMemProfiling"

**Comment by mikedoan** [comment]
Jenkins, test this please.

**Comment by mikedoan** [comment]
Jenkins, test this please.

**Comment by mikedoan** [comment]
Jenkins, test this please.

**Pull request merged**